release.py

```diff
@@ -446,9 +446,9 @@ def tweak_readme(tag: Tag, filename: str = "README.rst") -> None:
     print(f"Updating {filename}...", end=" ")
     readme = Path(filename)
 
-    # Update first line: "This is Python version 3.14.0 alpha 7"
+    # Update first line: "This is Python version X.Y.Z {release_level} N"
     # and update length of underline in second line to match.
-    lines = readme.read_text().splitlines()
+    lines = readme.read_text(encoding="utf-8").split("\n")
```

Comment:
```suggestion
    lines = readme.read_text().split("\n")
```

I think we can skip this, we have a platform check: 

https://github.com/python/release-tools/blob/a8981b1e639aa9ab69707cf33fb14850a809b807/run_release.py#L1375-L1384

And there's no RM using Windows now, or next. Perhaps the next one after Savannah will in 2028, but we can deal with it then (along with likely other Windows incompatibilities), and we might be using 3.15 as the minimum by then, with UTF-8 as default.